kivy/input/providers/mouse.py

```diff
@@ -330,45 +351,40 @@ def create_hover(self, win, etype):
     def on_mouse_motion(self, win, x, y, modifiers):
         nx, ny = win.to_normalized_pos(x, y)
         ny = 1.0 - ny
-        if self.current_drag:
-            touch = self.current_drag
-            touch.move([nx, ny])
-            touch.update_graphics(win)
-            self.waiting_event.append(('update', touch))
-        elif self.alt_touch is not None and 'alt' not in modifiers:
-            # alt just released ?
-            is_double_tap = 'shift' in modifiers
-            self.create_touch(win, nx, ny, is_double_tap, True, [])
+        for button in self.current_drag.keys():
+            if self.current_drag.get(button, None):
```

Comment:
The expression in this if-statement will always be True. Therefore the if-statement can be omitted.

Reply:
Thanks! good catch... fixed.